spec/model_spec.cr

```diff
@@ -97,4 +120,48 @@ describe LuckyRecord::Model do
       QueryMe.column_names.should eq [:id, :created_at, :updated_at, :email, :age]
     end
   end
+
+  describe ".ensure_correct_field_mappings" do
+    # table is missing
+    it "raises on missing table" do
+      missing_table = MissingTable.new(1, Time.new, Time.new)
+      expect_raises Exception, "The table uusers was not found, did you mean users?" do
+        missing_table.ensure_correct_field_mappings!
+      end
+    end
```

Comment:
I think another case for when the missing table doesn't have a similarly named table would also be good.